DBison.WPF/Controls/LineNumberedTextBox.cs

```diff
@@ -20,20 +37,120 @@ public override void OnApplyTemplate()
         }
         #endregion
 
+        #region [OnTextChanged]
+        protected override void OnTextChanged(TextChangedEventArgs e)
+        {
+            base.OnTextChanged(e);
+            if (!m_SkipHighliting)
+            {
+                __UpdateLineNumber();
```

Comment:
Sure that this should only happen when Highlighting is skipped? I'd expect the numbers to be always displayed.